src/flatpickr.js

```diff
@@ -1801,8 +1801,9 @@ function Flatpickr(element, config) {
 				.map(dObj => self.formatDate(dObj, self.config.altFormat))
 				.join(joinChar);
 		}
-
-		triggerEvent("ValueUpdate");
+		if (triggerChange !== false) {
```

Comment:
@zerkalica I see. but why not just use `onChange` then instead of `onValueUpdate`?